build.gradle.kts

```diff
@@ -184,7 +184,7 @@ publishing {
 }
 
 tasks.wrapper {
-    jarFile = file(".infra/gradle/gradle-wrapper.jar")
+    jarFile = file("gradle/wrapper/gradle-wrapper.jar")
```

Comment:
i think this is the default so you actually do not need to set it at all - actually we do not need the whole block, when we use default settings

Reply:
> i think this is the default so you actually do not need to set it at all - actually we do not need the whole block, when we use default settings

I don't know gradle very well but maybe this is needed for the option to use the gradle setting via a wrapper task. At least IntelliJ offers such an option beside `use default wrapper.properties location` and `use own location`.

Reply:
https://docs.gradle.org/current/dsl/org.gradle.api.tasks.wrapper.Wrapper.html#org.gradle.api.tasks.wrapper.Wrapper:jarFile

this is the default value, and the gradle wrapper task is something, which gradle provides out of the box since verison 5 i think (do not pin me on that). but you can find that information:

> Every vanilla Gradle build comes with a built-in task called wrapper. You’ll be able to find the task listed under the group "Build Setup tasks" when listing the tasks. Executing the wrapper task generates the necessary Wrapper files in the project directory. -  https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper

To test this, you can simply remove the block and run  `gradle wrapper` in your cli and see, it will work out of the box

Reply:
Ok removed the block and still seems to work. Thanks for the docs!